crates/node/src/storage/database/postgres.rs

```diff
@@ -653,6 +732,19 @@ mod tests {
 
     use super::*;
 
+    fn into_validated(tx: Transaction<TxCreate>) -> Transaction<TxValdiated> {
```

Comment:
Maybe implement `From` or `Into` ?

Reply:
Yes, as it was for a test, I didn't think to change the Tx API.
I do the change, but it fails because Transaction is declared in lib.rs and storage in main.rs so for the compiler it's 2 different crates and I get the error: `only traits defined in the current crate can be implemented for types defined outside of the crate`.
I put a TODO to do the change when the module declaration is corrected.